tests/Agency/StorePerformanceTest.cpp

```diff
@@ -0,0 +1,423 @@
+////////////////////////////////////////////////////////////////////////////////
+/// DISCLAIMER
+///
+/// Copyright 2014-2020 ArangoDB GmbH, Cologne, Germany
+/// Copyright 2004-2014 triAGENS GmbH, Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is ArangoDB GmbH, Cologne, Germany
+///
+/// @author Ignacio Rodriguez
+/// @author Copyright 2017, ArangoDB GmbH, Cologne, Germany
+////////////////////////////////////////////////////////////////////////////////
+
+#include "gtest/gtest.h"
+
+#include "Agency/Store.h"
+#include "Mocks/Servers.h"
+#include "fakeit.hpp"
+
+#include <velocypack/Builder.h>
+#include <velocypack/Compare.h>
+#include <velocypack/Slice.h>
+#include <velocypack/velocypack-aliases.h>
+
+#include <cstdlib>
+#include <algorithm>
+
+namespace arangodb {
+namespace tests {
+namespace store_performance_test {
+
+auto const scattered_times{50};
+auto const hammer_times {200};
```

Comment:
This is way too little for any meaningful measurement.

Reply:
I agree. This numbers were separated into constants specifically to be able to tune on our expectations. If these tests are to run with every Jenkins build, we still need to be cautious.